sklearn/base.py

```diff
@@ -290,10 +290,11 @@ def __repr__(self):
                                                offset=len(class_name),),)
 
     def __getstate__(self):
-        if type(self).__module__.startswith('sklearn.'):
```

Comment:
I think we still want this... we just want to delegate to `super` instead of blindly getting `self.__dict__.items()`.